DFC.ServiceTaxonomy.ApiFunction/Helpers/FileHelper.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace DFC.ServiceTaxonomy.ApiFunction.Helpers
+{
+    public class FileHelper : IFileHelper
+    {
+        public string ReadAllTextFromFile(string fileName)
+        {
+            return File.ReadAllText(fileName);
```

Comment:
We should consider using the async version `ReadAllTextAsync`, to (possibly) minimise the [possibility of thread exhaustion](https://docs.microsoft.com/en-us/azure/azure-functions/functions-best-practices). (Only possibly, as unsure if this ultimately results in an interop call to ReadFile, and whether that call blocks.)